liquibase-core/src/main/java/liquibase/ui/interactive/FileTypeEnum.java

```diff
@@ -0,0 +1,5 @@
+package liquibase.ui.interactive;
+
+public enum FileTypeEnum {
```

Comment:
This doesn't seem like it's a globally applicable enum. There are tons of file types we can deal with, and the list of what there CAN be is extensible. This should maybe be a nested enum somewhere instead of something top-level?